--- configure.wps	2020-07-16 13:22:20.189149760 +0000
+++ /shared/configure.wps	2020-07-16 06:13:31.317927007 +0000
@@ -57,12 +57,12 @@
 #   Settings for Linux x86_64, Intel compiler    (dmpar) 
 #
 #
-COMPRESSION_LIBS    = -L/shared/iccWRF/grib2/lib -ljasper -lpng -lz
-COMPRESSION_INC     = -I/shared/iccWRF/grib2/include
+COMPRESSION_LIBS    = -L/shared/gccWRF/grib2/lib -ljasper -lpng -lz
+COMPRESSION_INC     = -I/shared/gccWRF/grib2/include
 FDEFS               = -DUSE_JPEG2000 -DUSE_PNG
 SFC                 = ifort
 SCC                 = icc
-DM_FC               = mpif90 -f90=ifort
+DM_FC               = mpiifort
 DM_CC               = mpicc -cc=icc
 FC                  = $(DM_FC)
 CC                  = $(DM_CC)
@@ -71,8 +71,8 @@
 F77FLAGS            = -FI -convert big_endian
 FCSUFFIX            = 
 FNGFLAGS            = $(FFLAGS)
-LDFLAGS             = 
-CFLAGS              = -w
+LDFLAGS             =  -qopenmp
+CFLAGS              = -w -qopenmp
 CPP                 = /lib/cpp -P -traditional
 CPPFLAGS            = -D_UNDERSCORE -DBYTESWAP -DLINUX -DIO_NETCDF -DIO_BINARY -DIO_GRIB1 -DBIT32 -D_MPI
 ARFLAGS             =
