w--- makefile    2020-09-24 19:39:25.000000000 +0000
+++ ../makefile.new     2021-01-13 11:06:23.259686200 +0000
@@ -723,7 +723,7 @@
 all:   ${netcdf4src} ${hdf5src} ${prog} aux_progs/gmerge aux_progs/smallest_grib2 aux_progs/smallest_4
 
 
-${prog}:        $w/*.c $w/*.h ${jlib} ${aeclib} ${netcdf3lib} ${pnglib} ${hdf5lib} ${g2clib} ${netcdf4lib} ${iplib} ${spectrallib} ${gctpclib} ${proj4lib}
+${prog}:        $w/*.c $w/*.h ${aeclib} ${pnglib} ${g2clib} ${iplib} ${spectrallib} ${gctpclib} ${proj4lib}
        cd "$w" && export LDFLAGS="${wLDFLAGS}" && export CPPFLAGS="${wCPPFLAGS}" && ${MAKE}
 
 fast:        $w/*.c $w/*.h ${jlib} ${aeclib} ${netcdf3lib} ${pnglib} ${hdf5lib} ${g2clib} ${netcdf4lib} ${iplib} ${spectrallib} ${gctpclib} ${proj4lib}